### PR TITLE
👾 minor changes to dom node properties article 

### PR DESCRIPTION
**Pull request by Rnbsov**

### CODE REVIEW

File: 2-ui/1-document/05-basic-dom-node-properties/article.md [hunk 1]
```diff
@@ -18,16 +19,40 @@
...
+
+- [Node](https://dom.spec.whatwg.org/#interface-node) -- также является "абстрактным" классом, и служит основой для DOM-узлов.
+
+    Он обеспечивает базовую функциональность: `parentNode`, `nextSibling`, `childNodes` и т.д. (это геттеры). Объекты класса `Node` никогда не создаются. Но есть определённые классы узлов, которые наследуют от него (и следовательно наследуют функциональность `Node`).
+
+- [Document](https://dom.spec.whatwg.org/#interface-document), по историческим причинам часто наследуется `HTMLDocument` (хотя последняя спецификация этого не навязывает) -- это документ в целом.
+
+    Глобальный объект `document` принадлежит именно к этому классу. Он служит точкой входа в DOM.
+
+- [CharacterData](https://dom.spec.whatwg.org/#interface-characterdata) -- "абстрактным" класс, он наследуется:
```
Comment (ruslauz) [inline 2-ui/1-document/05-basic-dom-node-properties/article.md:34]: абстрактный

File: 2-ui/1-document/05-basic-dom-node-properties/article.md [hunk 2]
```diff
@@ -18,16 +19,40 @@
...
-- [HTMLElement](https://html.spec.whatwg.org/multipage/dom.html#htmlelement) -- является базовым классом для всех остальных HTML-элементов. От него наследуют конкретные элементы:
+- [EventTarget](https://dom.spec.whatwg.org/#eventtarget) -- это корневой "абстрактный" класс для всего.
+
+    Объекты этого класса никогда не создаются. Он служит основой, благодаря которой все DOM-узлы поддерживают так называемые "события", мы изучим их позже.
+
+- [Node](https://dom.spec.whatwg.org/#interface-node) -- также является "абстрактным" классом, и служит основой для DOM-узлов.
+
+    Он обеспечивает базовую функциональность: `parentNode`, `nextSibling`, `childNodes` и т.д. (это геттеры). Объекты класса `Node` никогда не создаются. Но есть определённые классы узлов, которые наследуют от него (и следовательно наследуют функциональность `Node`).
```
Comment (Alexandre887) [inline 2-ui/1-document/05-basic-dom-node-properties/article.md:28]:
```suggestion
    Он обеспечивает базовую функциональность: `parentNode`, `nextSibling`, `childNodes` и т.д. (это геттеры). Объекты класса `Node` никогда не создаются. Но есть определённые классы узлов, которые наследуются от него (и следовательно наследуют функционал `Node`).
```

File: 2-ui/1-document/05-basic-dom-node-properties/article.md [hunk 3]
```diff
@@ -18,16 +19,40 @@
...
+
+- [Document](https://dom.spec.whatwg.org/#interface-document), по историческим причинам часто наследуется `HTMLDocument` (хотя последняя спецификация этого не навязывает) -- это документ в целом.
+
+    Глобальный объект `document` принадлежит именно к этому классу. Он служит точкой входа в DOM.
+
+- [CharacterData](https://dom.spec.whatwg.org/#interface-characterdata) -- "абстрактный" класс, он наследуется:
```
Comment (Alexandre887) [inline 2-ui/1-document/05-basic-dom-node-properties/article.md:34]:
```suggestion
- [CharacterData](https://dom.spec.whatwg.org/#interface-characterdata) -- "абстрактный" класс. Вот, кем он наследуется:
```

File: 2-ui/1-document/05-basic-dom-node-properties/article.md [hunk 4]
```diff
@@ -18,16 +19,40 @@
...
+    - [Text](https://dom.spec.whatwg.org/#interface-text) -- класс, соответствующий тексту внутри элементов, например `Hello` в `<p>Hello</p>`.
```
Comment (Alexandre887) [inline 2-ui/1-document/05-basic-dom-node-properties/article.md:35]:
```suggestion
    - [Text](https://dom.spec.whatwg.org/#interface-text) -- класс, соответствующий тексту внутри элементов. Например, `Hello` в `<p>Hello</p>`.
```

File: 2-ui/1-document/05-basic-dom-node-properties/article.md [hunk 5]
```diff
@@ -18,16 +19,40 @@
...
+    - [Comment](https://dom.spec.whatwg.org/#interface-comment) -- класс для комментариев. Они не отображаются, но каждый комментарий становится членом DOM.
+
+- [Element](https://dom.spec.whatwg.org/#interface-element) -- это базовый класс для DOM-элементов.
+
+    Он обеспечивает навигацию на уровне элементов: `nextElementSibling`, `children` и методы поиска: `getElementsByTagName`, `querySelector`.
```
Comment (Alexandre887) [inline 2-ui/1-document/05-basic-dom-node-properties/article.md:40]:
```suggestion
    Он обеспечивает навигацию на уровне элементов: `nextElementSibling`, `children`.
    А также и методы поиска элементов: `getElementsByTagName`, `querySelector`.
```

File: 2-ui/1-document/05-basic-dom-node-properties/article.md [hunk 6]
```diff
@@ -18,16 +19,40 @@
...
+
+    Браузер поддерживает не только HTML, но также XML и SVG. Таким образом, класс `Element` служит основой для более специфичных классов: `SVGElement`, `XmlElement` (они нам здесь не нужны) и `HTMLElement`.
+
+- И наконец, [HTMLElement](https://html.spec.whatwg.org/multipage/dom.html#htmlelement) является базовым классом для всех остальных HTML-элементов. Мы будем работать с ним большую часть времени.
+
+    От него наследуют конкретные элементы:
```
Comment (Alexandre887) [inline 2-ui/1-document/05-basic-dom-node-properties/article.md:46]:
```suggestion
    От него наследуются конкретные элементы:
```

File: 2-ui/1-document/05-basic-dom-node-properties/article.md [hunk 7]
```diff
@@ -18,16 +19,40 @@
...
+
     - [HTMLInputElement](https://html.spec.whatwg.org/multipage/forms.html#htmlinputelement) -- класс для тега `<input>`,
     - [HTMLBodyElement](https://html.spec.whatwg.org/multipage/semantics.html#htmlbodyelement) -- класс для тега `<body>`,
     - [HTMLAnchorElement](https://html.spec.whatwg.org/multipage/semantics.html#htmlanchorelement) -- класс для тега `<a>`,
-    - ...и т.д, каждому тегу соответствует свой класс, который предоставляет определённые свойства и методы.
+    - ...и т.д.
+
+Существует множество других тегов со своими собственными классами, которые могут иметь определенные свойства и методы, в то время как некоторые элементы, такие как `<span>`, `<section>`, `<article>`, не имеют каких-либо определенных свойств, поэтому они являются экземплярами класса `HTMLElement`.
```
Comment (Alexandre887) [inline 2-ui/1-document/05-basic-dom-node-properties/article.md:53]:
```suggestion
Также существует множество других тегов со своими собственными классами, которые могут иметь определенные свойства и методы, в то время как некоторые элементы, такие как `<span>`, `<section>` и `<article>`, не имеют каких-либо определенных свойств, поэтому они являются экземплярами класса `HTMLElement`.
```

File: 2-ui/1-document/05-basic-dom-node-properties/article.md [hunk 8]
```diff
@@ -131,10 +156,10 @@ interface HTMLInputElement: HTMLElement {
   <script>  
   let elem = document.body;
 
-  // посмотрим что это?
+  // давайте проверим: какой тип узла находится в elem?
```
Comment (Alexandre887) [inline 2-ui/1-document/05-basic-dom-node-properties/article.md:159]:
```suggestion
  // давайте разберёмся: какой тип узла находится в elem?
```

**Review by Alexandre887 — CHANGES_REQUESTED** [review]

**Review by Alexandre887 — APPROVED** [review]

**Pull request merged**